sql/src/main/java/io/crate/statistics/TableStats.java

```diff
@@ -72,11 +72,7 @@ private long sizeInBytes(RelationName relationName) {
      * Returns -1 if the table isn't in the cache
      */
     public long estimatedSizePerRow(RelationName relationName) {
```

Comment:
Would it make sense to take the selected columns into considerations and utilize the ColumnStats if available? They'd provide more accurate size information if they're available

Reply:
I planned to do this separately